rust/ql/lib/codeql/rust/Concepts.qll

```diff
@@ -102,6 +102,58 @@ class ModeledEnvironmentSource extends EnvironmentSource::Range {
   ModeledEnvironmentSource() { sourceNode(this, "environment-source") }
 }
 
+/**
+ * A data flow source corresponding to a file access.
+ */
+final class FileSource = FileSource::Range;
```

Comment:
Aliases inherit the QL doc from the entities that they alias, so it is only needed when different QL doc is needed (same further down).